### PR TITLE
Add a border to the tree view

### LINKED ISSUE #8: Visual break between file finder and editor
Something I've noticed whilst using the UI and Syntax for the last few days in that there is no clear divide between the File Finder and the main editor. The two just blend together.

Just wondering if there would be some value in adding a subtle dividing line between the two? This should make it easier when it comes to resizing the file finder.

**Editor**
file-finder

There is a very subtle dividing line within the settings panel currently, its only subtle but it really helps divide the settings tabs and the content of each tab.

**Settings**
devide-in-settings

I love this UI and Syntax by the way, thanks for putting the time in to make this.

### PR DESCRIPTION
**Pull request by shaunbent**
Not sure if this is the right way to achieve this, I've never really played with Atom themes before.

**Before**
file-finder

**After**
border-added

This resolves #8

### CODE REVIEW

**Comment by Fequois** [comment]
Seems fine to me, but I'm no expert on Atom either!

I was wondering if there should be a border below the tab bar too? Maybe as an option? The tab bar floats a little too much for me, so I changed it in my local stylesheet, but perhaps more users would like the option.

**Comment by shaunbent** [comment]
I've added the same borders to the 'tabs' and the top of the 'find-and-replace' panel:

more-borders

I'm not sure about how we'd make this optional, in terms of adding to the options page and then linking it all up.

File: styles/tabs.less [hunk 1]
```diff
@@ -98,7 +99,7 @@
         &::after {
             content: '';
             position: absolute;
-            bottom: 0;
+            bottom: 1px;
```
Comment (shaunbent) [inline styles/tabs.less]: This is needed to keep the 2px border-bottom on the selected tab visible.

**Comment by Fequois** [comment]
That looks sweet to me, and thanks! Let's see what @silvestreh says.

**Comment by silvestreh** [comment]
I'll make this into an option as I'm not to fond of it. But if you guys want a border I'll give you that option :)

**Comment by Fequois** [comment]
Thanks a lot!

My eyes aren't what they used to be so I need visual cues. :wink:

**Comment by shaunbent** [comment]
@silvestreh cheers - I wasn't really sure about how to make it option or I'd have done it myself.

**Comment by silvestreh** [comment]
Would you rather have the entire tab-bar, tree-view, search-replace panels have a darker background or just a border?

**Comment by silvestreh** [comment]
@shaunbent also… may I ask which package/setting enables that sidebar with icons?

**Comment by Fequois** [comment]
@silvestreh It's `file-icons`.

Personally—darker colours, kind of like the One theme, but since it's not my PR or reported issue we'd better hear from Shaun first.

**Comment by silvestreh** [comment]
@Fequois I meant the other icons. Not the file-type icons :P
I'm more inclined to the darker background for the panels. I'm not a big fan of borders overall.

**Comment by Fequois** [comment]
Haha I noticed after replying, I sneakily removed that part of the comment. :smile:

**Comment by silvestreh** [comment]
Found it. It's called [tool-bar-main](https://atom.io/packages/tool-bar-main).

**Comment by Fequois** [comment]
Ah, knew I recognised it from somewhere: http://blog.atom.io/2015/05/21/new-package-roundup.html

**Comment by shaunbent** [comment]
I wouldn't mine darker backgrounds, just something that divides the panels.

Yeah tool-bar-main is the plugin. I thought it looked cool in the new package roundup so I installed it. Although I have taken it off today as I've had it for a month and never used it once.

**Comment by shaunbent** [comment]
If we're being true to 'Material Design' then it suggests using shadows to create layers:

https://www.google.com/design/spec/layout/principles.html#principles-how-paper-works

**Comment by silvestreh** [comment]
@shaunbent Yeah, I'm adding a few shadows.  I'm cleaning up some code and I'll push it in about an hour or less.

**Comment by silvestreh** [comment]
Just pushed 0.4.6. Should look like this:

captura de pantalla 2015-06-24 a las 14 44 36

Open a new issue if you run into any problems. I'm closing this Pull Request. 

Cheers!

**Comment by shaunbent** [comment]
Looks great - cheers for your help with this.

**Pull request closed without merging**